Libraries/src/Amazon.Lambda.RuntimeSupport/Client/InternalClientAdapted.cs

```diff
@@ -584,39 +584,8 @@ public async System.Threading.Tasks.Task<SwaggerResponse<StatusResponse>> ErrorW
             }
         }
 
-        private string ConvertToString(object value, System.Globalization.CultureInfo cultureInfo)
+        private string ConvertToString(string value, System.Globalization.CultureInfo cultureInfo)
         {
-            if (value is System.Enum)
```

Comment:
This code was not trimming compatible and it turn out our only use of the method was to the `value` being a string. So I choose to just simplify this method. Along time again this class was generated from a swagger file which I assume the generator just differentiated for all possible use cases.